projects/plugins/crm/src/automation/class-attribute-definition.php

```diff
@@ -0,0 +1,261 @@
+<?php
+/**
+ * Attribute Definition
+ *
+ * @package automattic/jetpack-crm
+ * @since $$next-version$$
+ */
+
+namespace Automattic\Jetpack\CRM\Automation;
+
+/**
+ * Step Attribute.
+ *
+ * The Step Attribute represents
+ *
+ * @since $$next-version$$
+ */
+class Attribute_Definition {
+
+	/**
+	* Represents a dropdown selection input.
+	*
+	* @since $$next-version$$
+	* @var string
+	*/
+	const SELECT = 'select';
+
+	/**
+	* Represents a checkbox input.
+	*
+	* @since $$next-version$$
+	* @var string
+	*/
+	const CHECKBOX = 'checkbox';
```

Comment:
Should these constants be a different class? Something like an enum of input types?

Reply:
Do you refer to the PHP enumeration? This was added from PHP 8, and we are compatible until PHP 7.2.

Reply:
Ah, in that case disregard this 😄

Reply:
👍